packages/react-components/src/components/VideoTile.tsx

```diff
@@ -86,6 +86,11 @@ export interface VideoTileProps {
    * Whether the video is muted or not.
    */
   isMuted?: boolean;
+  /* @conditional-compile-remove(pinned-participants) */
+  /**
+   * If true, the video tile will be rendered with a pin icon.
```

Comment:
Slightly misleading. This has nothing to do with the video.
If true, the VideoTile will show a pin icon.